packages/svelte/tests/runtime-runes/samples/spread-props/_config.js

```diff
@@ -1,6 +1,10 @@
 import { test } from '../../test';
 
 export default test({
+	compileOptions: {
+		dev: false
+	},
+
```

Comment:
I added comments to the other two but this seems like a legit bug?

```suggestion
```

Reply:
Yeah I think so too

Reply:
Ah my bad.